### PR TITLE
HTML Structure for Awesome Books

### PR DESCRIPTION
**Pull request by Qoosim**
We have added basic logic to create, add and remove books to the books array of objects and store the added books in the browser localStorage.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+// function to remove book
+function removeBook(element) {
+  if (element.classList.contains('remove')) {
+    const removeItem = element.parentElement;
+    books.splice(removeItem, 1);
+    removeItem.remove();
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
```
Comment (vichuge) [inline script.js]: - [x] When I tried to remove elements with diff author but same title, the application deletes the wrong book, please work on it. 🐶 

1. Try to follow the example on images, after deleting with the remove button, refresh the page and you'll see the bug.

image

Then:

image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Qoosim** [comment]
We resolved the error that was in the project in our first PR

File: script.js (hunk 1 above)
Comment (henatan99) [reply script.js]: This issue is well addressed. Nice job! :clap:

**Review by henatan99 — APPROVED** [review]
Hi @Qoosim , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [x] OPTIONAL: Kindly update your readme customizing to your project. A good readme makes your repo professional. This is a mandatory general requirement but since it was not pointed out by the previous reviewer I will make it optional, but please take it seriously. :pray: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**